GameServerLib/Packets/PacketHandlers/HandleSync.cs

```diff
@@ -35,7 +35,7 @@ public override bool HandlePacket(int userId, SynchVersionRequest req)
             }
             else
             {
-                _logger.Debug("Accepted client version (" + req.Version + ") from client="+req.ClientId);
+                _logger.Debug("Accepted client version (" + req.Version + ") from player="+userId);
```

Comment:
I honestly didn't think there was a point to keep it there, since the coun't will always be the same.

Just pushed the file with both ClientID and PlayerID.